number_of_1_bits.py

```diff
@@ -0,0 +1,3 @@
+class Solution:
+    def hammingWeight(self, n: int) -> int:

```

Comment:
I know that the name of this method was a given one from whatever platform you're using to train your algorithms, but at some point, you need to follow the best practices dictated in this case by PEP8 https://peps.python.org/pep-0008/

Reply:
method or functions should use underscores to split words